modules/table/src/main/java/org/apache/ignite/internal/table/AbstractTableView.java

```diff
@@ -76,7 +76,7 @@ protected IgniteException convertException(Throwable th) {
             return (IgniteException) th;
         }
 
-        //TODO: IGNITE-14500 Replace with public exception with an error code (or unwrap?).
+        //TODO: IGNITE-20181 KV/Binary view public API should only throw public exceptions for the end user.
```

Comment:
The change is not related to the current pull request - just a fix to a wrong issue link.